packages/nextjs/test/integration/next-env.d.ts

```diff
@@ -1,5 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
+/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
is this needed? just to clarify

Reply:
Nope, don't even know how this got in here... probably because I ran the tests locally. Will remove, thanks!